client/components/action-buttons/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+
+const ActionButtons = ( { buttons } ) => {
+	return (
+		<FormButtonsBar>
+			{ buttons.map( ( button, idx ) => (
+				<FormButton
+					type="button"
+					key={ idx }
+					disabled={ button.isDisabled }
+					onClick={ button.onClick }
+					isPrimary={ !! button.isPrimary }>
```

Comment:
We've foregone the use of `!!` for boolean casting in the past. Let's remove it.